task/rpm-ostree/0.1/rpm-ostree.yaml

```diff
@@ -0,0 +1,230 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  annotations:
+    tekton.dev/pipelines.minVersion: 0.12.1
+    tekton.dev/tags: image-build, appstudio, hacbs
+  labels:
+    app.kubernetes.io/version: "0.1"
+    build.appstudio.redhat.com/build_type: rpm-ostree
+    build.appstudio.redhat.com/multi-platform-required: "true"
+  name: rpm-ostree
+spec:
+  description: RPM Ostree
+  params:
+  - description: Reference of the image rpm-ostree will produce.
+    name: IMAGE
+    type: string
+  - default: quay.io/sdouglas/rpm-ostree:dev
```

Comment:
Assuming this is temporary

Reply:
Oops, I had been testing with a RHTAP built image in redhat-user-workloads, and missed the default was still personal. I will update it to the redhat-user-workloads one, but I can't just build on GH actions like other images as it needs to be multi platform image>

Reply:
Could be built in RHTAP eventually?